src/com/komiamiko/fcorbit/CommandNone.java

```diff
@@ -73,7 +73,7 @@ public void mouseReleased(MouseEvent e) {
 					view.repaint();
 				}
 			}else{
-				int mx = e.getX(), my = e.getY();
+				int mx = view.originMousex, my = view.originMousey;
```

Comment:
Tracks the origin of the mouseDown event when selecting an object.

Reply:
I know. That's the point. Without this change, one might click down, move the mouse slightly, and then get an unexpected result when they release. This way, it's based on where they clicked.